target.go

```diff
@@ -124,6 +139,8 @@ func (b *Basic) start() {
 			err := b.w.Write(rec)
 			if err != nil {
 				rec.Logger().Logr().ReportError(err)
```

Comment:
Maybe we should register the target errors like we're doing with the loggedcount for logr and target. What do you think? (maybe I missed something)

Reply:
Thanks @ethervoid , that's a good idea.  I added drop count and block count as well.